app/controllers/provider_interface/deferred_offer/navigation.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
```

Comment:
Pedantry warning, we don't need this anymore. I think there is a rubcop rule that we can enable...?